stellar/utils.js

```diff
@@ -340,6 +340,14 @@ function stellarAddressToBytes(address) {
     return hexlify(Buffer.from(address, 'ascii'));
 }
 
+function isValidAddress(address) {
+    try {
+        return !!Address.fromString(address);
+    } catch {
+        return false;
+    }
```

Comment:
```suggestion
    try {
        // try conversion
        Address.fromString(address);
        return true;
    } catch {
        return false;
    }
```
make it more readable